coaster/src/frameworks/cuda/device.rs

```diff
@@ -125,20 +125,20 @@ pub struct DeviceInfo {
     info: Vec<u8>,
 }
 
-impl DeviceInfo {
+impl std::fmt::Display for DeviceInfo {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(f, "{}", unsafe { String::from_utf8_unchecked((*self.info).to_owned()) })
+    }
+}
 
```

Comment:
Since this is fail able, maybe use the parse fn or another safe fun that returns a result